src/Organizer/Events/LabelsImported.php

```diff
@@ -53,11 +64,16 @@ public static function deserialize(array $data): LabelsImported
     public function serialize(): array
```

Comment:
I don't think it's needed to change this method. The internal property is still a VO `Labels`. You can also directly access this property and don't need to use the `getLabels` accessor method. So I reverted this change.